amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -1903,19 +1904,15 @@ VX_API_ENTRY vx_node VX_API_CALL vxExtrppNode_CropMirrorNormalize(vx_graph graph
             (vx_reference)pSrc,
             (vx_reference)srcROI,
             (vx_reference)pDst,
-            (vx_reference)dstROI,
```

Comment:
dstRoI is being removed here. Please check if this works as expected

Reply:
Yes it was removed intentionally